WordPress/src/main/java/org/wordpress/android/ui/posts/PostUtils.java

```diff
@@ -545,6 +545,9 @@ public static String getPreviewUrlForPost(RemotePreviewType remotePreviewType, P
                 break;
             case REMOTE_PREVIEW_WITH_REMOTE_AUTO_SAVE:
                 previewUrl = post.getAutoSavePreviewUrl();
+                if (UrlUtils.isHomePage(previewUrl)) {
```

Comment:
@antonis I'm just now catching up on your progress here. Nice work tracking down this fix! I haven't tested this out at all but I'm wondering from your research, would there be any negatives effects if we include the slug in all previews, not just Home Page? 

I don't have any reason to make the change outside of I think that helps a bit with the "I feel that this is just a hack." feeling.

Reply:
Tbh I didn't think of this. I tried it now (on the browser) and it seems that appending the slug to a non-home page url (that already has a slug) makes no difference since a redirection occurs. E.g.
`https://antonistest.wordpress.com/redhill/redhill/?preview_id=831&preview_nonce=d50d43d924` redirects to
`https://antonistest.wordpress.com/redhill/?preview_id=831&preview_nonce=d50d43d924`

>I don't have any reason to make the change outside of I think that helps a bit with the "I feel that this is just a hack." feeling.

I think that's the only downside :|

Reply:
Ah with having it to a redirect hop that might not be ideal then since we know we can avoid it by only doing it for the special case. Thanks for investigating this! I'm good with only appending the slug for Home pages then.